### ISSUE TITLE
"No such file or directory" error on adding reference to csproj file. 

### ISSUE THREAD

**Issue by hungriestPigeon** [waf/CSharpRepl#27]
### Version

C# REPL 0.3.0 and 0.2.9 on dotnet 5.0.302 on Ubuntu 20.10

### What happened?

Trying to add a reference to my .csproj file produced the error message "No such file or directory". Trying to reference members from the project anyway fails with "error CS0103: The name {name} does not exist in the current context". 

Minimal example:

```
$ dotnet new console -o myApp
The template "Console Application" was created successfully.

Processing post-creation actions...
Running 'dotnet restore' on myApp/myApp.csproj...
  Determining projects to restore...
  Restored ~/csharp/hello/myApp/myApp.csproj (in 87 ms).
Restore succeeded.

$ cd myApp
$ dotnet run
Hello World!
$ ls
bin  myApp.csproj  obj  Program.cs
$ csharprepl -r myApp.csproj
Welcome to the C# REPL (Read Eval Print Loop)!
Type C# expressions and statements at the prompt and press Enter to evaluate them.
Type help to learn more, and type exit to quit.

Adding supplied references...
Building myApp.csproj
No such file or directory
> #r "myApp.csproj"
Building myApp.csproj
No such file or directory
> Main()
(1,1): error CS0103: The name 'Main' does not exist in the current context
> Program.Main()
(1,1): error CS0103: The name 'Program' does not exist in the current context
> myApp.Main()  
(1,1): error CS0103: The name 'myApp' does not exist in the current context
> myApp.Program.Main()
(1,1): error CS0103: The name 'myApp' does not exist in the current context

```

Using the full path to the file does not solve the problem.

**Comment by waf** [comment]
Thanks for reporting! In addition to fixing this, sounds like it's time to set up some Ubuntu integration tests.

I'll try to get the fix out over the weekend.

**Comment by hungriestPigeon** [comment]
Thanks waf. Don't rush on my account though, I have now figured out that referencing the dll file works fine, so I can do everything I need to.

**Comment by waf** [comment]
This is fixed, but not published yet. When I've published the fix I'll update this issue.

**Comment by waf** [comment]
This fix has been released as part of 0.3.1. You can get it with `dotnet tool update -g csharprepl`
